docker-compose-3-persona.yml

```diff
@@ -121,20 +124,23 @@ services:
       npx @digicatapult/dscp-process-management@latest create -h hydrogen-producer-node -p 9944 -u //Alice -f ./processFlows.json &&
       npm start"
     environment:
-      - PORT=8000
-      - LOG_LEVEL=debug
-      - NODE_HOST=hydrogen-producer-node
-      - NODE_PORT=9944
-      - DB_HOST=hydrogen-producer-postgres-hyproof-api
-      - DB_PORT=5432
-      - DB_NAME=dscp-hyproof-api
-      - DB_USERNAME=postgres
-      - DB_PASSWORD=postgres
-      - IDENTITY_SERVICE_HOST=hydrogen-producer-identity
-      - IDENTITY_SERVICE_PORT=9000
-      - IPFS_HOST=ipfs
-      - IPFS_PORT=5001
-      - USER_URI=//Alice
+      - 'PORT=8000'
+      - 'LOG_LEVEL=debug'
+      - 'NODE_HOST=hydrogen-producer-node'
+      - 'NODE_PORT=9944'
+      - 'DB_HOST=hydrogen-producer-postgres-hyproof-api'
+      - 'DB_PORT=5432'
+      - 'DB_NAME=dscp-hyproof-api'
+      - 'DB_USERNAME=postgres'
+      - 'DB_PASSWORD=postgres'
+      - 'IDENTITY_SERVICE_HOST=hydrogen-producer-identity'
+      - 'IDENTITY_SERVICE_PORT=9000'
+      - 'IPFS_HOST=ipfs'
+      - 'IPFS_PORT=5001'
+      - 'USER_URI=//Alice'
+      - 'API_SWAGGER_BG_COLOR=#e4f2f3'
+      - 'API_SWAGGER_TITLE=API-Heidi'
```

Comment:
Again I think we should get the word `HyProof` in here so maybe something like `Hyproof API - Heidi`